Makefile

```diff
@@ -22,6 +24,10 @@ push-container:
 manifests:
 	cd manifests && IMG_REPOSITORY=${IMG_REPOSITORY} IMG_TAG=${IMG_TAG} envsubst < kustomization.yaml.in > kustomization.yaml
 
+.PHONY: kustomize
+kustomize: manifests
+	kustomize build ${MANIFETS_PATH} > ./manifests/${PROJECT_NAME}.yaml
```

Comment:
Can you name it something like `manifests-release` instead of `kustomize`?
Use `oc kustomize` instead of `kustomize build`, otherwise we would need to download the kustomize binary. It may not be installed.
Can you output the bundle yaml somewhere outside of `./manifests` directory? We don't want to `git commit` it into the repository, so it would be clearer if it is not in a directory that is usually committed.

Reply:
What will we use then in GitHub Action? we don't have a step to setup `oc`.

Reply:
Right, I didn't realize that. Then we can create a new make target to download the `kustomize` binary and use it instead of `oc`. Similarly to how it is done in ssp-operator: 

https://github.com/kubevirt/ssp-operator/blob/1120a9cc277b4dcad19ac14cbc438b8aeced2d55/Makefile#L231-L235